examples/field_cu/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: 2020 CERN
+# SPDX-License-Identifier: Apache-2.0
+
+set(ADEPT_COMMON_SRCS
+  field_step.cu
+)
+
+# add field_step.cu
+foreach(SRC_FILE ${ADEPT_COMMON_SRCS})
+  get_filename_component(SRC_FILENAME ${SRC_FILE} NAME_WE)
+
+  set(SRC_FILE_CPP ${CMAKE_CURRENT_SOURCE_DIR}/${SRC_FILE})
+  set(SRC_FILE_CU ${CMAKE_CURRENT_BINARY_DIR}/cuda_src/${SRC_FILENAME}.cu)
```

Comment:
This shouldn't be needed, the file already has the extensions `.cu` so `nvcc` is happy

Reply:
Thanks.